### DISCUSSION TITLE
Stellar Quest Chapter 1 - by Nev

### DISCUSSION THREAD

**Discussion by BLAKKMASS** [stellar/sorobanathon#170] [General]
My submission for Chapter 1 of Stellar Quest
- Nev (@BLAKKMASSART on twitter)

https://litemint.com/items/GBBFRN6ZP4TIPQ3E3YKMBZP332BJXMYISQK365IQX43H5WUGRXOGYH6F/NFT

![NevStellarQuest](https://user-images.githubusercontent.com/120704944/208033578-aff245f4-e96f-42e0-afd3-ceb6c54cdac8.jpg)

**Comment by stellarjohn** [comment]
Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 300 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John
